driver/tab_completion.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import importlib.util
+from pathlib import Path
+import subprocess
+import sys
+
+from . import util
+
+try:
+    import argcomplete
+    HAS_ARGCOMPLETE = True
+except ImportError:
+    HAS_ARGCOMPLETE = False
+
+
+def complete_build_arg(prefix, parsed_args, **kwargs):
+    if parsed_args.debug:
+        argcomplete.warn("The option --debug is an alias for --build=debug. Do no specify both --debug and --build.")
+        exit(1)
+
+    builds_folder = Path(util.REPO_ROOT_DIR) / "builds"
+    if not builds_folder.exists():
+        argcomplete.warn("No build exists.")
```

Comment:
We don't know what to check for at this point: we might just need the translator and it would be fine if there is no search binary (or the other way around). We later check for the tool we need and complain if it is not there.

Reply:
I just meant if no folders (like "release" or "debug") exist within the "builds" folder. But it's fine as it is.

Reply:
"Check for existence, then used" (look before you leap, LBYL)  is considered an anti-pattern in Python (and not just in Python) because the folder might exist now but be gone by the time we try to access it. The preferred style is to attempt the access and handle an exception if it doesn't work. See for example here:

https://realpython.com/python-lbyl-vs-eafp/#working-with-files-and-directories